docs/deployment/components.md

```diff
@@ -0,0 +1,58 @@
+---
+myst:
+  html_meta:
+    "description": "Components of a Plone 6 application for deployment"
+    "property=og:description": "Components of a Plone 6 application for deployment"
+    "property=og:title": "Components of a Plone 6 application for deployment"
+    "keywords": "Plone, deployment, components, backend, volto, fronted, TLS termination proxy, load balancer, router, database, mail service, optimization, maintenance"
+---
+
+(deployment-components-label)=
+
+# Components
+
+This page in the deployment guide covers the components in a Plone application deployment.
+The components can be broken down into those that are required, recommended, and optional.
+
+
+(deployment-required-components-label)=
+
+## Required components
+
+You need the following components to deploy a Plone application.
+
+-   {term}`Plone backend`
+-   {term}`Volto` as the default {term}`Frontend` for Plone
+-   {term}`TLS termination proxy`
+-   {term}`Load balancer` or router
+-   Database
+-   Mail service
+-   Optimization tasks
+    -   Database parameters
+    -   https://5.docs.plone.org/manage/deploying/performance/index.html
+-   Maintenance tasks
+    -   Database backup
+    -   Packing database - https://5.docs.plone.org/manage/deploying/packing.html
+    -   Monitoring
+    -   Log rotation
+    -   Firewall
+    -   Attack protection
+    -   DNS
+    -   Host name resolution
```

Comment:
Everything from firewall on down is not a maintenance task, but an area of concern that should be addressed when setting up the underlying system that hosts Plone.

I think these are mostly out of scope for the Plone docs, except:
- we can say that you ought to think about them
- in the server-enviroment chapter, we'll show one way to address them

It might make sense to move mentioning them from here to server-environment.

Also, aren't DNS and host name resolution the same thing?

Reply:
I can't remember the term where you configure the web server. Is that "listen on IP address" or "bind" or something else?

Reply:
Mostly addressed in 9d29c455f12282530f29200a467394da0f80e4b5, but might need some polishing for the web server configuration bit.

Reply:
@davisagli please let me know if this change is OK. I'd like to merge this PR soon to replace the current Deployment skeleton.